src/trimStart.js

```diff
@@ -0,0 +1,29 @@
+import { invoker } from 'ramda';
+
+import polyfill from './internal/polyfills/String.trimStart';
+import isFunction from './isFunction';
+
+export const trimStartPolyfill = polyfill;
+
+export const trimStartInvoker = invoker(0, 'trimStart');
```

Comment:
No need to export invoker here

Reply:
Taking this back. This is consistent with other part of the codebase (replaceAll)